src/sentry/models/sentryfunction.py

```diff
@@ -4,6 +4,12 @@
 from sentry.db.models.fields.array import ArrayField
 
 
+class SentryFunctionManager(BaseManager):
+    def get_sentry_functions(self, organization_id, event_type):
+        functions = self.filter(organization_id=organization_id, events__contains=event_type)
+        return list(functions)
```

Comment:
Do you need to list cast? Having a queryset allows you to add additional conditions and defer query execution until you need to use the results. Fetching results also happens automatically when you iterate a queryset.